src/main/java/org/folio/services/acquisitions/AcquisitionMethodService.java

```diff
@@ -39,12 +37,12 @@ public void createAcquisitionsMethod(AcquisitionMethod acquisitionMethod, Contex
       .onSuccess(entity -> {
         log.info("AcquisitionMethod with id {} created", acquisitionMethod.getId());
         asyncResultHandler.handle(Future.succeededFuture(
-          OrdersStorageAcquisitionMethods.PostOrdersStorageAcquisitionMethodsResponse
-            .respond201WithApplicationJson(entity, headersFor201())));
+          PostOrdersStorageAcquisitionMethodsResponse.respond201WithApplicationJson(
+            entity, PostOrdersStorageAcquisitionMethodsResponse.headersFor201())));
```

Comment:
If there is no any business logic changes here, I propose to revert these import changes

Reply:
The old version looked inconsistence: it used two methods from the same class:
`OrdersStorageAcquisitionMethods.PostOrdersStorageAcquisitionMethodsResponse.respond201WithApplicationJson` without import and `headersFor201` with static import. I'll rollback if that was intended

Reply:
Lets remain this change, because
OrdersStorageAcquisitionMethods.PostOrdersStorageAcquisitionMethodsResponse.respond201WithApplicationJson  - very hard to read.
also I propose to remain static import for headersFor201() to simplify reading as well